src/codegen/functions/function.py

```diff
@@ -9,12 +9,16 @@ class Function:
 
     def __init__(self, function_name, function_metadata):
         """Structure for storing function metadata from scrapigen."""
+        if "python_description" in function_metadata:
+            self._description = function_metadata["python_description"]
         self._function_name = function_name
-        self._description = function_metadata["python_description"]
         self._is_python_factory = function_metadata.get("is_python_factory", False)
-        self._python_class_name = function_metadata["python_class_name"]
+        if "python_class_name" in function_metadata:
+            self._python_class_name = function_metadata["python_class_name"]
         self._calling_convention = function_metadata["calling_convention"]
         self._return_type = function_metadata["returns"]
+        if "stream_response" in function_metadata:
+            self._stream_response = function_metadata["stream_response"]
```

Comment:
This is a bool and it should default to False, so use `function_metadata.get("stream_response", False)`

Reply:
Updated